app.js

```diff
@@ -3,108 +3,34 @@
 var counter = 0;
 var questionName = 'What is your name?';
 var userName = prompt(questionName);
+console.log(questionName + ': ' + userName);
 
-function question(){
-  console.log(questionName + ': ' + userName);
-  alert('Howdy ' + userName + '!');
-};
-question();
+var questionPrompt = ['Do you want to play a game of memory?', 'Is my last name MacCoy?', 'Did I attend two colleges?', 'Did I mention I\'m back in school again?', 'Is my nickname Dingo?'];
+var responseYes = ['Superb! Hopefully you read my bio!', 'Incorrect, it\'s McCoy, attention to detail is important!', 'Bingo! Technically 3 but you didn\'t know that.', 'Correct! You must have gone to school for being so smart.', 'Correct! You must know voodoo because I never mentioned that.'];
+var responseNo = ['Sad panda, come back later when you\'re not scared.', 'Who ever thought answering \'no\' could feel so right.', 'Wrong; maybe alzheimers is setting in.', 'Wah wah; I would see a dr. for your memory.', 'Wrong but I won\'t fault you because I never mentioned it.'];
+var userScore = [[1,0],[0,1],[1,0],[1,0],[1,0]];
 
 function question1(){
-  var question1 = 'Do you want to play a game of memory?';
-  var answer1 = prompt(question1);
-  if (answer1.toLowerCase() === 'y' || answer1.toLowerCase() === 'yes') {
-    var responseYes1 = alert('Superb! Hopefully you read my bio!');
-    console.log(question1 + ': ' + answer1);
-    counter++;
-  }
-  else if (answer1.toLowerCase() === 'n' || answer1.toLowerCase() === 'no') {
-    var responseNo1 = alert('Sad panda, come back later when you\'re not scared.');
-    console.log(question1 + ': ' + answer1);
-  }
-  else {
-    var responseNA1 = alert('Woah there, tiger; try using english.');
-    console.log(question1 + ': ' + answer1);
+  for (var i = 0; i < questionPrompt.length; i++) {
```

Comment:
this is great.  nice job bringing all of these questions into one reusable chunk of logic.